who_is_active.sql

```diff
@@ -2897,6 +3074,21 @@ BEGIN;
 								AND s.last_request_end_time <= sp.last_request_end_time
 							)
 						)
+					LEFT JOIN sys.dm_exec_query_stats AS qs ON

```

Comment:
- Check formatting on these joins.
- I think you don't need to join to query_stats on both the start and end offsets - start should be enough.

Reply:
I will blend this in with the rest of the changes on my next commit

Reply:
Ok, ready for a review, I hope I didn't miss anything.